explicit-any plugins doesn't work

Min repo setup: https://github.com/lkonya/ts-migrate-test-repo
Result of running `npx -p ts-migrate -c "ts-migrate-full src"` can be found on https://github.com/lkonya/ts-migrate-test-repo/tree/ts-migrate.

```
Your default tsc path is ./node_modules/.bin/tsc.

[Step 1 of 4] Initializing ts-config for the "src"...

Config file created at /Users/laszlokonya/WebstormProjects/ts-migrate/src/tsconfig.json
[master 5e604ee] [ts-migrate][src] Init tsconfig.json file
 1 file changed, 71 insertions(+)
 create mode 100644 src/tsconfig.json
/Users/laszlokonya/WebstormProjects/ts-migrate

[Step 2 of 4] Renaming files from JS/JSX to TS/TSX and updating project.json\...

Renaming 1 JS/JSX files in /Users/laszlokonya/WebstormProjects/ts-migrate/src...
Done.
[master bfaaa3a] [ts-migrate][src] Rename files from JS/JSX to TS/TSX
 1 file changed, 0 insertions(+), 0 deletions(-)
 rename src/{index.js => index.ts} (100%)
/Users/laszlokonya/WebstormProjects/ts-migrate

[Step 3 of 4] Fixing TypeScript errors...

TypeScript version: 4.1.2
Initialized tsserver project in 597ms.
Start...
[strip-ts-ignore] Plugin 1 of 14. Start...
[strip-ts-ignore] Finished in 10ms.
[hoist-class-statics] Plugin 2 of 14. Start...
[hoist-class-statics] Finished in 1ms.
[react-props] Plugin 3 of 14. Start...
[react-props] Finished in 1ms.
[react-class-state] Plugin 4 of 14. Start...
[react-class-state] Finished in 1ms.
[react-class-lifecycle-methods] Plugin 5 of 14. Start...
[react-class-lifecycle-methods] Finished in 1ms.
[react-default-props] Plugin 6 of 14. Start...
[react-default-props] Finished in 1ms.
[react-shape] Plugin 7 of 14. Start...
[react-shape] Finished in 1ms.
[declare-missing-class-properties] Plugin 8 of 14. Start...
[declare-missing-class-properties] Finished in 211ms.
[member-accessibility] Plugin 9 of 14. Start...
[member-accessibility] Finished in 1ms.
[explicit-any] Plugin 10 of 14. Start...
[explicit-any] Finished in 24ms.
[add-conversions] Plugin 11 of 14. Start...
[add-conversions] Finished in 13ms.
[eslint-fix] Plugin 12 of 14. Start...
Error occurred in eslint-fix plugin :(
[eslint-fix] Finished in 5ms.
[ts-ignore] Plugin 13 of 14. Start...
[ts-ignore] Finished in 3ms.
[eslint-fix] Plugin 14 of 14. Start...
Error occurred in eslint-fix plugin :(
[eslint-fix] Finished in 1ms.
Finished in 272ms, for 14 plugin(s).
Writing 1 updated file(s)...
Wrote 1 updated file(s) in 2ms.
[master a8c2731] [ts-migrate][src] Run TS Migrate
 1 file changed, 2 insertions(+)
/Users/laszlokonya/WebstormProjects/ts-migrate

[Step 4 of 4] Checking for TS compilation errors (there shouldn't be any).

./node_modules/.bin/tsc -p src/tsconfig.json
src/index.ts:8:35 - error TS7006: Parameter 'req' implicitly has an 'any' type.

8 const server = http.createServer((req, res) => {
                                    ~~~

src/index.ts:8:40 - error TS7006: Parameter 'res' implicitly has an 'any' type.

8 const server = http.createServer((req, res) => {
                                         ~~~

src/index.ts:18:25 - error TS7006: Parameter 'req' implicitly has an 'any' type.

18 function randomFunction(req, res) {
                           ~~~

src/index.ts:18:30 - error TS7006: Parameter 'res' implicitly has an 'any' type.

18 function randomFunction(req, res) {
                                ~~~

src/index.ts:24:14 - error TS7006: Parameter 'str' implicitly has an 'any' type.

24 function log(str){
                ~~~


Found 5 errors.
```

Function parameters doesn't get `: any` type. 

Any suggestions what should I do? I used ts-migrate 2 weeks ago, and it added `: any` everywhere, but now it doesn't work. What do i miss?

Env:
OS: MacOS 11.2.1
shell: fish 3.2.0
node: 14.16.0
ts-migrate: 0.16

--- a/packages/ts-migrate-server/src/migrate/index.ts
+++ b/packages/ts-migrate-server/src/migrate/index.ts
@@ -50,9 +50,6 @@ export default async function migrate({
   log.info('Start...');
   const pluginsTimer = new PerfTimer();
   const updatedSourceFiles = new Set<string>();
-  const sourceFiles = project
-    .getSourceFiles()
-    .filter(({ fileName }) => !/(\.d\.ts|\.json)$|node_modules/.test(fileName));
 
   for (let i = 0; i < config.plugins.length; i += 1) {
     const { plugin, options: pluginOptions } = config.plugins[i];
@@ -60,6 +57,10 @@ export default async function migrate({
     const pluginLogPrefix = `[${plugin.name}]`;
     const pluginTimer = new PerfTimer();
     log.info(`${pluginLogPrefix} Plugin ${i + 1} of ${config.plugins.length}. Start...`);
+
+    const sourceFiles = project
+      .getSourceFiles()
+      .filter(({ fileName }) => !/(\.d\.ts|\.json)$|node_modules/.test(fileName));
 
     // eslint-disable-next-line no-restricted-syntax
     for (const sourceFile of sourceFiles) {

--- a/packages/ts-migrate-server/tests/commands/migrate/migrate.test.ts
+++ b/packages/ts-migrate-server/tests/commands/migrate/migrate.test.ts
@@ -44,4 +44,41 @@ describe('migrate command', () => {
     expect(rootData).toEqual(outputData);
     expect(exitCode).toBe(0);
   });
+
+  it('Migrates project with two plugins', async () => {
+    const inputDir = path.resolve(__dirname, 'input');
+    const outputDir = path.resolve(__dirname, 'output_two');
+    const configDir = path.resolve(__dirname, 'config');
+
+    copyDir(inputDir, rootDir);
+    copyDir(configDir, rootDir);
+
+    const config = new MigrateConfig()
+      .addPlugin(
+        {
+          name: 'test-plugin-1',
+          run({ text }) {
+            const newText = text.replace('test string', 'updated string');
+            return newText;
+          },
+        },
+        {},
+      )
+      .addPlugin(
+        {
+          name: 'test-plugin-2',
+          run({ text }) {
+            const newText = text.replace('updated string', 'another updated string');
+            return newText;
+          },
+        },
+        {},
+      );
+
+    const exitCode = await migrate({ rootDir, config });
+    fs.unlinkSync(path.resolve(rootDir, 'tsconfig.json'));
+    const [rootData, outputData] = getDirData(rootDir, outputDir);
+    expect(rootData).toEqual(outputData);
+    expect(exitCode).toBe(0);
+  });
 });

--- a/packages/ts-migrate-server/tests/commands/migrate/output_two/index.ts
+++ b/packages/ts-migrate-server/tests/commands/migrate/output_two/index.ts
@@ -1,0 +1,1 @@
+console.log('another updated string');

